PubnativeLite/PubnativeLiteDemo/Adapters/AppMonet Adapters/DFP Header Bidding Adapter/AppMonet+DFP.m

```diff
@@ -21,31 +21,93 @@
 //
 
 #import "AppMonet+DFP.h"
+#import <HyBid/HyBid.h>
+#import "AppMonetConfigurations.h"
 #import "HyBidLogger.h"
 
 @import GoogleMobileAds;
 
+@interface AppMonet ()
+
+@property (class, nonatomic, strong) HyBidAdRequest *adRequest;
+@property (class, nonatomic, strong) HyBidInterstitialAdRequest *interstitialAdRequest;
+@property (class, nonatomic) NSMutableDictionary *dict;
+@property (class, nonatomic) NSMutableDictionary *interstitialDict;
+@property (class, nonatomic, assign) BOOL isDFP;
+
+typedef struct {
```

Comment:
I guess here's you'll want to apply the same changes @faresbh  did 'cause there seemed to be an access issue when using the struct from a static method.

Reply:
yes, dictionaries and requests should change here as in MoPub.